src/widget/gui.cpp

```diff
@@ -322,25 +322,34 @@ QString GUI::passwordDialog(const QString& cancel, const QString& body)
 
 void GUI::_clearContacts()
 {
-    Nexus::getDesktopGUI()->clearContactsList();
+    Widget* w = Nexus::getDesktopGUI();
+    if (w)
+        w->clearContactsList();
```

Comment:
Is that really a requirement these days? I took a quick look at the code and the extra braces seems to be used very inconsistently.
Methinks it would be more acceptable if we were using K&R braces, but with Allman it seems a bit excessive in vertical space used.